include/linux/kfence.h

```diff
@@ -21,44 +23,23 @@ void kfence_cache_unregister(struct kmem_cache *s);
 
 bool kfence_discard_slab(struct kmem_cache *s, struct page *page);
 
-size_t kfence_ksize(void *object);
+size_t kfence_ksize(const void *object);
 
 bool kfence_handle_page_fault(unsigned long address);
 
-#else
-static inline void kfence_init(void)
-{
-}
-static inline void *kfence_alloc_and_fix_freelist(struct kmem_cache *s)
-{
-	return NULL;
-}
+#else /* CONFIG_KFENCE */
+
+static inline void kfence_init(void) { }
+static inline void *kfence_alloc_and_fix_freelist(struct kmem_cache *s) { return NULL; }
```

Comment:
Did clang-format do this change? Otherwise we'll lose the formatting next time we apply the tool.

Reply:
No it didn't. This is what the formatting should look like for these in the end. I think it's not too much of an issue to manually format the header file. Alternatively we could put

// clang-format off

here, and remove when we're done. I'll go with // clang-format off.